### PR TITLE
fixed arguments passed to top function

### PR DESCRIPTION
**Pull request by agilefall**

### CODE REVIEW

File: signal_analog/flow.py [hunk 1]
```diff
@@ -379,6 +379,8 @@ def __str__(self):
             str_arg = str(self.arg)
         return "%s=%s" % (self.keyword, str_arg)
 
+    def __eq__(self, other):
```
Comment (agilefall) [inline signal_analog/flow.py]: This is only used of testing.  Do we want this?

File: signal_analog/flow.py [hunk 2]
```diff
@@ -181,9 +181,9 @@ def sum(self, by=None, over=None):
         self.call_stack.append(Sum(by=by, over=over))
         return self
 
-    def top(self, by=None, over=None):
+    def top(self, count=None, percentage=None, by=None):
```
Comment (agilefall) [inline signal_analog/flow.py]: This isn't backwards compatible.  Should we create a 2.x release branch so we can get all of the api changes in one release?

File: tests/test_signal_analog_flow.py [hunk 3]
```diff
@@ -52,6 +52,12 @@ def test_find_label_published():
 
     assert program.find_label('A') == data
 
+def test_top_function():
```
Comment (agilefall) [inline tests/test_signal_analog_flow.py]: Not sure how to test this.  This is very yucky though.

**Comment by agilefall** [comment]
The lint-ing failed, but there warnings/errors from all sorts of files I didn't touch

File: signal_analog/flow.py (hunk 2 above)
Comment (dogonthehorizon) [reply signal_analog/flow.py]: Great suggestion, I think that makes sense.

File: signal_analog/flow.py (hunk 1 above)
Comment (dogonthehorizon) [reply signal_analog/flow.py]: I don't see why not.

**Comment by agilefall** [comment]
I updated the PR to point to the 2.0 branch

**Comment by dogonthehorizon** [comment]
My focus is to get a draft going for https://github.com/Nike-Inc/signal_analog/issues/47 so that we can have a plan for rolling this change out in a timely manner.

**Pull request merged**